pkg/cluster/internal/create/create.go

```diff
@@ -115,6 +116,10 @@ func Cluster(logger log.Logger, p providers.Provider, opts *ClusterOptions) erro
 		actionsToRun = append(actionsToRun,
 			kubeadminit.NewAction(opts.Config), // run kubeadm init
 		)
+		// Crafting: patch kube-proxy
+		if crafting.InSandboxWorkspace() {
+			actionsToRun = append(actionsToRun, crafting.NewKubeProxyPatchAction(p, opts.Config))
+		}
```

Comment:
Let's try to avoid breaking the abstraction. I suggest add additional logic to `nodeCmd.Run`. If the command is `kubeadm init`, in `nodeCmd.Run` after it completes, apply the patches.